src/Profile/InfoBoxConfig.cpp

```diff
@@ -127,6 +127,10 @@ Profile::Load(const ProfileMap &map, InfoBoxSettings &settings)
     break;
   }
 
+  map.Get(ProfileKeys::InfoBoxTitleScale, settings.scale_title_font);
+  if (settings.scale_title_font < 50 || settings.scale_title_font > 150)
+    settings.scale_title_font = 100; // discard unreasonable values
```

Comment:
Why discard?
I'll repeat my question: did you mean to clamp instead?

Reply:
settings.scale_title_font comes from the active *.prf . This file may contain just any value. For instance if the user changed it in a text editor.
The default for settings.scale_title_font is 100, corresponding to 100%.
I might as well clamp, but I thought using the default value in such a (rare) case would be more appropriate.